### PR TITLE
Write changelog in one style

### PR DESCRIPTION
**Pull request by Leyn-pl**
It was hard (name 0.6 and zone)

### CODE REVIEW

**Review by Slotterleet — CHANGES_REQUESTED** [review]
v0.6, v.0.6.1 and v1.0 don't have any specific names.
New zone in 0.6 was Abandoned Desert.

File: changelog.txt [hunk 1]
```diff
@@ -32,32 +32,51 @@ Fixes:
...
+New items:
+    - Research Point
+    - Shiny Alloy
+New units:
+    - Ghost Miner Drone (suggested by [hb] Kapzduke)
+    - Golden Eradicator
+New mechs:
+    - Dart Mk. II
+New blocks:
+    - Gost Mined Drone Factory
```
Comment (Slotterleet) [inline changelog.txt:45]: Ghost Miner Drone Factory

Comment (Leyn-pl) [reply changelog.txt:45]: Mined bruh

Comment (Slotterleet) [reply changelog.txt:45]: If nothing changes in 24 hours, this PR will be closed.

**Pull request closed without merging**